### DISCUSSION TITLE
How to extract 'Structural Analysis' from nested dictionary ??

### DISCUSSION THREAD

**Discussion by AlamgirKhan-GIS** [giswqs/geog-312#195] [Q&A] [answered]
![Screenshot 2024-12-17 190006](https://github.com/user-attachments/assets/3104cf6f-fcff-41d6-a99b-20bffa3d2ab5)

**Comment by giswqs** [comment, marked as answer]
`['Name']`

**Reply by AlamgirKhan-GIS** [reply]
![Uploading Screenshot 2024-12-17 203309.png…]()
Thank you Professor!
